src/components/AdminDashboard/NotificationsDrawer.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  Drawer,
+  DrawerBody,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  useDisclosure,
+  IconButton,
+  Button,
+  Box,
+  Text,
+  VStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Badge,
+  useToast,
+} from '@chakra-ui/react';
+import { BellIcon, ChevronDownIcon, ArrowForwardIcon } from '@chakra-ui/icons';
+import { useState } from 'react';
+
+const NotificationsDrawer = ({ notificationsData }) => {
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'notificationsData' is missing in props validation